emsdk.py

```diff
@@ -2767,6 +2767,8 @@ def expand_sdk_name(name, activating):
           backend = 'fastcomp'
       return 'sdk-releases-%s-%s-64bit' % (backend, release_hash)
     elif len(version) == 40:
+      if backend is None:
+        backend = 'upstream'
```

Comment:
Just set `backend = 'upstream'` when it initialized on ling 2748?

Reply:
We can't do that, as we need to track if the user provided a backend. If not, we autodetect it based on the version on lines 2764-7. That code would break with your proposed change.